src/openaq/incremental_downloader_parallel.py

```
@@ -481,4 +497,4 @@ def download_country_all(self, country_code: str, country_id: int,
         print(f"Output file: {output_path}")
         print(f"{'='*60}")
 
-        return str(output_path)
+        return str(output_path)
```

Pylint (3.3.7)

[convention] Final newline missing

(C0304)